.github/workflows/build-deploy-ocr.yml

```diff
@@ -54,14 +56,14 @@ jobs:
           docker-pw: ${{ secrets.GITHUB_TOKEN }}
           docker-username: ${{ github.actor }}
           docker-tag: ${{ steps.set_tag.outputs.docker_tag }}
-          dockerfile-path: ./OCR/Dockerfile
-          docker-context-path: ./OCR/
+          dockerfile-path: ./backend/Dockerfile
+          docker-context-path: ./backend/
           api-name: ocr
 
   deploy-ocr:
     name: Deploy OCR
     runs-on: ubuntu-latest
-    environment: ${{ inputs.deploy-env }}
+    environment: dev
```

Comment:
Do we want to hardcode `dev` here?  Or can we make dynamic so if whomever picks it up can select `dev`, as well as, `stage` or `prod` (after they build out those environments)?

Reply:
this is how we do local github action testing. I will make this a draft

Reply:
also wouldnt need to like this if we create a seperate worflow for building and deploying the middleware